### PR TITLE
add the negative connections

### PR DESCRIPTION
**Pull request by etuleu**
I am not sure about these so please review carefully @rcoreilly

### CODE REVIEW

File: axon/pvlv_net.go [hunk 1]
```diff
@@ -607,21 +614,40 @@ func (net *Network) AddPVLVOFCus(ctx *Context, nUSneg, nYneur, popY, popX, bgY,
...
+		"Prjn.PrjnScale.Abs": "2", // strong
+		"Prjn.PrjnScale.Rel": ".2",
+	}
 	pj = net.ConnectToMatrix(blaPosAcq, vSmtxGo, p1to1)
 	pj.DefParams = params.Params{
 		"Prjn.PrjnScale.Abs": "4", // key strength driver
 		"Prjn.PrjnScale.Rel": "1",
 	}
 	pj.SetClass("BLAAcqToGo")
+	pj = net.ConnectToMatrix(blaNegAcq, vSmtxGo, p1to1)
```
Comment (rcoreilly) [inline axon/pvlv_net.go:628]: p1to1 -> full

File: axon/pvlv_net.go [hunk 2]
```diff
@@ -607,21 +614,40 @@ func (net *Network) AddPVLVOFCus(ctx *Context, nUSneg, nYneur, popY, popX, bgY,
...
+	net.ConnectLayers(blaNegAcq, vSstnp, full, ForwardPrjn)
+	net.ConnectLayers(blaNegAcq, vSstns, full, ForwardPrjn)
 
 	pj = net.ConnectToMatrix(blaPosExt, vSmtxNo, p1to1)
 	pj.DefParams = params.Params{
 		"Prjn.PrjnScale.Abs": "0.1", // extinction is mostly within BLA
 		"Prjn.PrjnScale.Rel": "1",
 	}
 	pj.SetClass("BLAExtToNo")
+	pj = net.ConnectToMatrix(blaNegExt, vSmtxNo, p1to1)
```
Comment (rcoreilly) [inline axon/pvlv_net.go:645]: p1to1 -> full

**Review by rcoreilly — CHANGES_REQUESTED** [review]
Logically, there is no connection between positive and negative USs.  VsMatrix and OFCus are all organized around positive USs.  Therefore, it only makes sense to have full projections from Neg to these layers.  Can you give that a try?

Also, don't forget to run boa TestStdCPUnData1 with SaveStandard = true to update test results after updating something like this, to pass the tests.

**Comment by etuleu** [comment]
> Logically, there is no connection between positive and negative USs. VsMatrix and OFCus are all organized around positive USs. Therefore, it only makes sense to have full projections from Neg to these layers. Can you give that a try?
> 
> Also, don't forget to run boa TestStdCPUnData1 with SaveStandard = true to update test results after updating something like this, to pass the tests.

Made the changes. Take a look again please.

**Review by rcoreilly — APPROVED** [review]
more to do but good enough for now!

**Pull request merged**